lib/platform/gitlab/index.ts

```diff
@@ -99,7 +99,17 @@ export async function initPlatform({
 export async function getRepos(): Promise<string[]> {
   logger.debug('Autodiscovering GitLab repositories');
   try {
-    const url = `projects?membership=true&per_page=100&with_merge_requests_enabled=true&min_access_level=30`;
+    const query = new URLSearchParams({
+      membership: 'true',
+      per_page: '100',
+      with_merge_requests_enabled: 'true',
+      min_access_level: '30',
+    });
+    const lastActivityAfter = process.env.GITLAB_LAST_ACTIVITY_AFTER;
```

Comment:
Instead of using a custom env var we should use a config option for self-hosted renovate users. This options can passed on initPlatform.

Reply:
Shall we call it `autodiscoverSortLastActivity` with type boolean? Then we either need to implement it for each platform (preferred) or document it as GitLab-only (less preferred).